pkg/cmd/roachtest/tests/backup_fixtures.go

```diff
@@ -228,70 +230,83 @@ func (bd *backupDriver) monitorBackups(ctx context.Context) {
 func registerBackupFixtures(r registry.Registry) {
 	for _, bf := range []backupFixtureSpecs{
 		{
-			// Default AWS Backup Fixture
-			hardware:            makeHardwareSpecs(hardwareSpecs{workloadNode: true}),
-			backup:              makeBackupFixtureSpecs(scheduledBackupSpecs{}),
-			timeout:             5 * time.Hour,
-			initFromBackupSpecs: backupSpecs{version: "v22.2.0"},
-			skip:                "only for fixture generation",
+			// 400GB backup fixture with 48 incremental layers. This is used by
+			// - restore/tpce/400GB/aws/inc-count=48/nodes=4/cpus=8
+			// - restore/tpce/400GB/aws/nodes=4/cpus=16
+			// - restore/tpce/400GB/aws/nodes=4/cpus=8
+			// - restore/tpce/400GB/aws/nodes=8/cpus=8
+			hardware:             makeHardwareSpecs(hardwareSpecs{workloadNode: true}),
+			scheduledBackupSpecs: makeBackupFixtureSpecs(scheduledBackupSpecs{}),
+			timeout:              5 * time.Hour,
+			initWorkloadViaRestore: restoreSpecs{
+				backup:                 backupSpecs{version: "v22.2.0", numBackupsInChain: 48},
+				restoreUptoIncremental: 48,
```

Comment:
why do we need to specify `restoreUpToIncremental` during fixture generation?

Reply:
hrm, I think `numBackupsInChain` identifies the bucket:
https://github.com/cockroachdb/cockroach/blob/691b544bbe40675b620e62a3505b8c7a32a5c825/pkg/cmd/roachtest/tests/restore.go#L688

Reply:
we can try deleting this `restoreUpToIncremental`  set in a follow up

Reply:
oh the String() method of `restoreUpToIncremental`, but I think this is used to pick out the AOST I believe - https://github.com/cockroachdb/cockroach/blob/cca37917931978b791d241e877ba3f55f6cdaea3/pkg/cmd/roachtest/tests/restore.go#L701? I imagine we want all 48 incs to actually have the size of data that we claim?

Reply:
ahhhh, I see. Ok no problems in your patch. Thanks for doing this!